### ISSUE TITLE
[REQUEST]KeyError: 'bpy_prop_collection[key]: key "Subdivision" not found'

### ISSUE THREAD

**Issue by lilili-lilili** [princeton-vl/infinigen#122]
Optional services are not configured during Infinigen configuration，
When implementing the basic example in the second step you provided, an error will occur when performing the following steps
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task coarse -g desert.gin simple.gin --output_folder outputs/helloworld/coarse

There are three errors：
[times] [INFO] | [Create terrain] failed with <class 'KeyError'>
[times] [INFO] | [MAIN TOTAL] failed with <class 'KeyError'>
KeyError: 'bpy_prop_collection[key]: key "Subdivision" not found'

**Comment by araistrick** [comment]
Hello, could you elaborate what you mean by 'optional services'? Also, please re-attempt this command using 1.0.3, as it seems there has been some kind of error here such that the system didnt think a scenetype config was loaded.

**Comment by xiaolin-mt** [comment]
I ran into the same issue and found this is caused by the system language used by the install Blender. You can launch Blender and change the language from Chinese to English to solution this issue.

**Comment by araistrick** [comment]
Interesting. Did blender use Chinese by default or did it stop working after manually switching the language? Short term I can add a check so that the error message is at least more understandable. Long term we may be able abstract the code so that it works either way, although I will have to check there exists a good mechanism for this via the blender API.

**Comment by xiaolin-mt** [comment]
Yes, Blender uses Chinese by default. I don't know why this is happening, given the system language of the OS is also English. No error occurred after I switched from Chinese to English, and I successfully built the HelloWorld demo. I assume there may be an option with Blender when building to configure the default language of Blender.
